addons/mrp/models/mrp_production.py

```diff
@@ -1922,7 +1922,6 @@ def button_mark_done(self):
         for production in self:
             production.write({
                 'date_finished': fields.Datetime.now(),
-                'product_qty': production.qty_produced,
```

Comment:
Pay attention the "cost structure" report use `product_qty` as "produced quantity"